hoot-core/src/main/cpp/hoot/core/io/db/SqlBulkInsert.cpp

```diff
@@ -78,8 +78,12 @@ inline QString SqlBulkInsert::_escape(const QVariant& v) const
   case QVariant::String:
     {
       QString result = v.toString();
-      result.replace("'", "''");
-      return "'" % result % "'";
+      if (!result.contains("hstore(ARRAY", Qt::CaseInsensitive) && result != "''") //check tags string return from HootApiDb::_escapeTags(tags)
+      {
+         result.replace("'", "''");
```

Comment:
This prevents inserting sql error if the string itself contains a single quote, for example: "ming's value". Escaping single quotes ' by doubling them up -> ''